pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -307,6 +308,67 @@ public void overrideDefaultValuesIfKeyIsPresentInConfigFile() throws IOException
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
 
+  @Test
+  public void permissionsSmartContractWithoutOptionMustDisplayUsage() {
+    parseCommand("--permissions-nodes-contract-address");
+
+    verifyZeroInteractions(mockRunnerBuilder);
+
+    assertThat(commandErrorOutput.toString())
+        .startsWith("Missing required parameter for option '--permissions-nodes-contract-address'");
+    assertThat(commandOutput.toString()).isEmpty();
+  }
+
+  @Test
+  public void permissionsEnabledWithoutContractAddressMustNotError() {
```

Comment:
I believe we want the CLI to fail when the user enables smart contract based permissioning but doesn't specify the contract address. We can't do smart contract permissioning without the address.

Reply:
agreed

Reply:
this test now errors